### PR TITLE
Use caporalJS

### PR DESCRIPTION
**Pull request by ArifAhmed1995**
@mukkachaitanya Review please.

### CODE REVIEW

**Comment by mukkachaitanya** [comment]
@ArifAhmed1995 I don't see you have updated the `package.json` file with the new dependency, `caporal`. I would suggest you to do so.

**Comment by ArifAhmed1995** [comment]
@mukkachaitanya Done. Also do change the `description` and `name` to whatever suits best.

**Comment by mukkachaitanya** [comment]
@ArifAhmed1995 can you rebase with master to pull the latest change.

File: index.js [hunk 1]
```diff
@@ -1,63 +1,64 @@
...
-	process.exit(1);
-}
+prog
+    .version('1.0.0')
+    .description('Program to facilitate pairwise evaluation in CS F213 Labs')
+    .command('pair_eval', 'Compute pairwise evaluation scores.')
+    .argument('<teamcsv>', 'CSV Data for Teams')
+    .argument('<scorescsv>', 'CSV Data for Scores')
+    .action(function(args, options, logger) {
+        if(Object.keys(args).length != 2){
```
Comment (mukkachaitanya) [inline index.js]: Can you expose this functionality via a different file, say `main.js` and then separate caporal and the functionality itself.

**Review by mukkachaitanya — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -1,63 +1,64 @@
...
+	        logger.error("Usage: node index.js pair_eval <path_to_team.csv_file> <path_to_scores.csv_file>\n\nOutputs a teamScores.csv file to the current directory");
+        }
 
-var teamCSV = process.argv[2];
-var scoresCSV = process.argv[3];
+        const promiseMarks = input.csvContents(args.teamcsv);
```
Comment (mukkachaitanya) [inline index.js]: Also the files are reversed here. Change: 
```
const promiseMarks = input.csvContents(args.scorescsv)
const promiseTeam = input.csvContents(args.teamcsv);
```

**Review by mukkachaitanya — CHANGES_REQUESTED** [review]

**Comment by mukkachaitanya** [comment]
@ArifAhmed1995 Also once you implement modularity. Could you write a test for the same?

**Comment by ArifAhmed1995** [comment]
I guess for the test we can have an optional argument for the `computeTeamResults` function. If that value is undefined then final result is written to current directory, otherwise is written to the filepath passed. For the test that value would refer to a `tempFile` which will be discarded after test(as already done in the test for `output`).

**Review by mukkachaitanya — APPROVED** [review]

**Comment by mukkachaitanya** [comment]
Everything looks fine here. Merging.

**Pull request merged**